### PR TITLE
Release v3.14.2

### PR DESCRIPTION
**Pull request by Akarshit**
# v3.14.2
This release is coordinated with the release of [reaction-admin v3.0.0-beta.22](https://github.com/reactioncommerce/reaction-admin/releases/tag/v3.0.0-beta.22), [reaction v3.13.4](https://github.com/reactioncommerce/reaction/releases/tag/v3.13.4), [example-storefront v4.1.2](https://github.com/reactioncommerce/example-storefront/releases/tag/v4.1.2), [reaction-identity v3.3.1](https://github.com/reactioncommerce/reaction-identity/releases/tag/v3.3.1) to keep the  up-to-date with the latest version of all our development platform projects.

### CODE REVIEW

File: config/reaction-oss/reaction-v3.14.2.mk [hunk 1]
```diff
@@ -0,0 +1,34 @@
+###############################################################################
+### Reaction OSS v3.11.1
```
Comment (manueldelreal) [inline config/reaction-oss/reaction-v3.14.2.mk]: I think this is a byproduct of a copy. Let's update the version number to reflect the overall version change.

**Review by delagroove — APPROVED** [review]

**Review by manueldelreal — COMMENTED** [review]
version is mismatched

File: config/reaction-oss/reaction-v3.14.2.mk (hunk 1 above)
Comment (Akarshit) [reply config/reaction-oss/reaction-v3.14.2.mk]: Oh nice one. I will fix this.

**Review by manueldelreal — APPROVED** [review]
:shipit:

**Pull request merged**